neuland/CMakeLists.txt

```diff
@@ -31,5 +31,5 @@ endif(WITH_EPICS)
 add_subdirectory(reconstruction)
 add_subdirectory(shared)
 add_subdirectory(simulation)
-add_subdirectory(test)
+#add_subdirectory(test)
```

Comment:
why this line is commented?

Reply:
Sorry, I had at first a very old version of R3BRoot and had to comment this for it to run, never changed it back.